spec/e2e/enum_spec.rb

```diff
@@ -160,4 +160,23 @@
       end.to raise_error(Neo4j::Shared::Enum::ConflictingEnumMethodError)
     end
   end
+
+  context 'when using `ActionController::Parameters`' do
+    require 'rack'
+    require 'rack/test'
+    require 'action_controller/metal/strong_parameters'
+    let(:params) { ActionController::Parameters.new('type' => 'image') }
```

Comment:
Maybe we could create a helper function like `action_controller_params(hash)` which does the appropriate `require`s to clean things up?